packages/bits/demo/src/components/demo/table/table-virtual-scroll-sticky-header/table-virtual-scroll-sticky-header-example.component.html

```diff
@@ -11,15 +11,29 @@
 
         <!-- IMPORTANT Bind only visible rows to the table, otherwise, the table will render all rows in the DOM -->
         <table nui-table [dataSource]="visibleItems$ | async" [trackBy]="trackByNo" [sortable]="true">
-            <!--         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^  -->
-            <ng-container *ngFor="let column of displayedColumns" [nuiColumnDef]="column">
-                <th nui-header-cell *nuiHeaderCellDef> {{column}}</th>
-                <td nui-cell *nuiCellDef="let element">{{element[column]}}</td>
+            <!-- ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ -->
+            <ng-container *ngFor="let column of displayedColumns" [nuiColumnDef]="column" [type]="column === 'icon' ? 'icon' : 'default'">
+                <th nui-header-cell *nuiHeaderCellDef>
+                    <span *ngIf="column !== 'icon'; else headerIcon">{{column}}</span>
+                    <ng-template #headerIcon>
+                        <div class="d-flex align-items-center justify-content-center">
+                            <nui-icon icon="enable"></nui-icon>
+                        </div>
+                    </ng-template>
+                </th>
+                <td nui-cell *nuiCellDef="let element">
+                    <span *ngIf="column !== 'icon'; else icon">{{element[column]}}</span>
+                    <ng-template #icon>
+                        <div class="d-flex align-items-center justify-content-center">
+                            <nui-icon [icon]="element[column]"></nui-icon>
+                        </div>
+                    </ng-template>
+                </td>
```

Comment:
I used this documentation example for proof that it's fixed since we didn't have a virtual scroll table in the examples with an icon column. I thought it might be useful for consumers. Let me know if you think I should revert this example and create a separate test component.

Reply:
Scratch that. The angel on my right shoulder requested that I create a separate test component to keep this one as simple as possible.